src/ethereum/cancun/vm/instructions/system.py

```diff
@@ -515,17 +515,24 @@ def selfdestruct(evm: Evm) -> None:
     beneficiary_balance = get_account(evm.env.state, beneficiary).balance
     originator_balance = get_account(evm.env.state, originator).balance
 
-    # First Transfer to beneficiary
-    set_account_balance(
-        evm.env.state, beneficiary, beneficiary_balance + originator_balance
-    )
-    # Next, Zero the balance of the address being deleted (must come after
-    # sending to beneficiary in case the contract named itself as the
-    # beneficiary).
-    set_account_balance(evm.env.state, originator, U256(0))
+    if (
+        originator in evm.env.state._created_accounts
```

Comment:
If `_created_accounts` is going to be accessed outside the module in which it was defined, we should drop the leading underscore.

Reply:
Updated